scripts/apps/ingest/views/settings/service-config.html

```diff
@@ -22,6 +22,16 @@
         <div sd-ingest-config-errors></div>
     </div>
 
+    <div class="form__row" ng-if="field.type === 'url_request'" ng-show="isConfigFieldVisible(field)">
+        <button
+            type="button"
+            class="btn btn--primary"
+            ng-click="doUrlRequest(field)">
+            {{field.label | translate}}
```

Comment:
`| translate` should only be applied to static strings. Otherwise, the string can't be statically extracted for translation. You should remove the `translate` filter here, and translate the static string before it's assigned to a variable.

Reply:
Hey @tomaskikutis I was surprised too, but it's used like this all over the place in this file. If the word is translated it should work, the only thing which can't work is the marking.

Reply:
Yep, it's used everywhere in this file, but it shouldn't be. Since you are editing the file already, it would be good if you could fix all instances.

It's an unstable solution to have to rely on other strings to be translated in order for this to work. If this label got renamed, for example, it would break, and the new string would not get extracted. It's quite easy to fix too.

Reply:
_posting back this comment which was lost in some weird parallel universe_:

it should be marked in the backed though, but the issue is that I don't think we have a mechanism to match backend translation language to user preference in client.

Reply:
> I don't think we have a mechanism to match backend translation language to user preference in client.

That's a good opportunity to fix back-end translations then. User language can be taken from `user.language` property.

Reply:
well, that's definitely not something I'll do with this issue, it's a whole new thing.

Reply:
as an update, after talking with Petr it seems that we have a solution for that, so I've translated the labels in the backend, and I will update this patch soon.

Not sure about the other labels of the files, I suspect that I would need to check all services, including customer ones, so that's probably work for a separate ticket. I'll have a look though.

Reply:
fixed in this PR, I've created https://dev.sourcefabric.org/browse/SDESK-5431 for other labels.